indico/web/assets/bundles.py

```diff
@@ -406,8 +406,8 @@ def rjs_bundle(name, *files):
     'contributions': rjs_bundle('modules_contributions',
                                 *namespace('js/indico/modules/contributions', 'common.js', 'userlists.js')),
     'timetable': rjs_bundle('modules_timetable',
-                            *namespace('js/indico/modules/timetable/timetable', 'Base.js', 'DragAndDrop.js', 'Draw.js',
-                                       'Filter.js', 'Layout.js', 'Management.js', 'Undo.js')),
+                            *namespace('js/indico/modules/timetable/timetable', 'Filter.js', 'Layout.js', 'Undo.js',
```

Comment:
What's the reason for this change? Seems wrong that `Base.js` is not the first one...